README.md

```diff
@@ -1 +1,53 @@
-# aws-connect-sso-web-app
\ No newline at end of file
+# AWS Connect SSO Web App
+
+This project is intended to run from AWS Amplify, using Cognito as the log in
+mechanism with users configured to have specialized groups representing their
+Call Center permissions for various AWS Connect instances. This works in concert
+with the Lambda defined for
+[generating SAML Responses for Connect](https://github.com/newjersey/custom-aws-idp).
+Then, once a user logs in to Cognito, this site should help them (within zero to
+two clicks) get federated and logged in to their desired Connect instance.
+
+## Configuration
+
+This site relies on some environment variables:
+
+- `NEXT_PUBLIC_SAML_GENERATION_URL`
+
+  The URL for the APIGateway connection to the SAML Response generating Lambda
+  mentioned above
+
+- `NEXT_PUBLIC_COGNITO_HOSTED_UI_URL`
+
+  The base URL for the Hosted UI Cognito sign-in page associated with this
+  UserPool's application (the current working UserPool is "_Cognito for Connect
+  Call Centers_" (ID: _us-east-1*jKQHCtx7s_) and the "App integration" client is
```

Comment:
This should be `us-east-1_jKQHCtx7s`, right?

Reply:
Good catch!